tests/plugins/uploadimage/manual/imagecustomtypes.html

```diff
@@ -0,0 +1,15 @@
+<div style="display: none; text-align: center; color: red; font-weight: bold; padding: 20px;" id="ignore">
+	File uploading is not supported on Internet Explorer 9 and below.
+</div>
+<div id="editor1"></div>
+<script>
+	CKEDITOR.replace( 'editor1', {
+		height: 400,
+        uploadUrl: '%BASE_PATH%',
+        uploadImage_supportedTypes: [ 'png' ]
+	} );
+
+	if ( CKEDITOR.env.ie && CKEDITOR.env.version < 10 ) {
```

Comment:
We have [`CKEDITOR.plugins.fileTools.isFileUploadSupported` method](https://github.com/ckeditor/ckeditor-dev/blob/master/plugins/filetools/plugin.js#L869-L882) (not released yet, but already on master). It seems like a better fit for such feature detection than browser sniffing.